examples/transfer_position.py

```diff
@@ -1,7 +1,242 @@
+        )
+        instruments = instruments_response.json()["result"]
+        instrument = next(
+            inst for inst in instruments if inst["instrument_name"] == instrument_name
+        )
+    else:
+        instruments_response = requests.post(
+            "https://api-demo.lyra.finance/public/get_instruments",
+            json={"currency": "ETH", "instrument_type": "option", "expired": False},
+            headers={"accept": "application/json", "content-type": "application/json"},
+        )
+        instruments = instruments_response.json()["result"]
+        instrument = next(
+            inst for inst in instruments if inst["instrument_name"] == instrument_name
+        )
+
+    print(f"Selected instrument for transfer: {instrument['instrument_name']}")
+
+    # Get position details for dynamic amount and price
+    position = first_position
+    position_amount = (
+        Decimal(position["amount"]) if position["amount"] != "0" else Decimal("0.1")
+    )  # Use 0.1 as minimum if no position
+    position_price = (
+        Decimal(position["mark_value"])
+        if position["mark_value"] != "0"
+        else Decimal(instrument.get("mark_price", "100"))
+    )
+
+    position_amount = position_amount.quantize(Decimal("0.1"), rounding=ROUND_HALF_UP)
+    position_price = position_price.quantize(Decimal("0.1"), rounding=ROUND_HALF_UP)
+
+    print(f"Position amount: {position_amount}")
+    print(f"Position average price: {position_price}")
+
+    ###################
+    # Define Transfer #
+    ###################
+
+    # Transfer amount and price from position data, with 0 fees for transfers
+    transfer_amount = abs(position_amount)  # Use absolute value for transfer amount
+    transfer_price = position_price
+    max_fee = Decimal("0")  # No fees for transfers
+
+    # Determine maker direction - maker must reduce their position
+    # If position is positive, maker sells (reduces long position)
+    # If position is negative, maker buys (reduces short position)
+    original_position_amount = Decimal(position["amount"])
+    if original_position_amount > 0:
+        maker_is_bid = False  # Sell to reduce long position
+        maker_direction = "sell"
+        taker_is_bid = True  # Taker buys
+        taker_direction = "buy"
+    else:
+        maker_is_bid = True  # Buy to reduce short position
+        maker_direction = "buy"
+        taker_is_bid = False  # Taker sells
+        taker_direction = "sell"
+
+    print(f"Original position amount: {original_position_amount}")
+    print(f"Maker direction: {maker_direction} (reducing position)")
+    print(f"Taker direction: {taker_direction}")
+
+    print("Creating transfer-specific signed actions...")
+
+    # Create maker order parameters
+    maker_nonce = utils.get_action_nonce()
+    maker_signature_expiry = utils.MAX_INT_32
+
+    # Create taker order parameters - ensure different nonce
+    time.sleep(0.001)  # Small delay to ensure different timestamp
```

Comment:
Note, if you just sign after you make the first action no need to pollute with time.sleep. 

Less imports and less code == better